### PR TITLE
Make it clearer that user can manually enter name, phone, address in Send screen  

### LINKED ISSUE #225: Make it clearer that a user can manually enter/send to a phone number or address
### Desired Behavior

Update experience when entering recipient information. 

Changes to Search Field:
- Magnifying glass icon has been replaced with “To”
- Search field placeholder copy updated

Changes to Recipient items:
- When user enters a phone number, update the label to say "Send to Phone Number"
- When user enters an address, update the label to say "Send to Address"

Designs: [Figma](https://www.figma.com/file/zt7aTQ5wuXycIwxq5oAmF9/Wallet-Refresh?node-id=8203%3A5819)

Screen Shot 2021-04-09 at 5 14 42 PM

### PR DESCRIPTION
**Pull request by michellewong793**
### Description

Changes: 
Magnifying glass icon in Send input replaced with “To”
Search field placeholder copy updated
When user enters a phone number, update the label to say "Send to Phone Number"
When user enters an address, update the label to say "Send to Address"

image
image

### Tested

Tested on Android and iOS 

### Related issues

- Fixes #225

### CODE REVIEW

File: packages/react-components/components/WithPasteAware.tsx [hunk 1]
```diff
@@ -9,17 +9,20 @@ interface PasteAwareProps {
...
   isPasteIconVisible: boolean
   onPressPaste: () => void
   onChangeText: (text: string) => void
+  leftIcon?: React.ReactNode
 }
 
 interface PasteAwareState {
   isPasteIconVisible: boolean
   clipboardContent: string | null
+  leftIcon?: React.ReactNode
```
Comment (gnardini) [inline packages/react-components/components/WithPasteAware.tsx:25]: The `PasteAwareState` doesn't need a `leftIcon`, it's just used in the props :)

**Review by gnardini — COMMENTED** [review]
Great job! Left some minor comments and we're good to go :)

File: packages/mobile/src/send/Send.tsx [hunk 2]
```diff
@@ -333,7 +333,10 @@ class Send extends React.Component<Props, State> {
       // needs fullscreen rendering
       <View style={styles.body}>
         <DisconnectBanner />
-        <SendSearchInput onChangeText={this.onSearchQueryChanged} />
+        <SendSearchInput
+          onChangeText={this.onSearchQueryChanged}
+          leftIcon={<Text style={styles.leftIcon}>{i18n.t('global:to')}</Text>}
```
[the remark this answers is not in the document]
Comment (michellewong793) [reply packages/mobile/src/send/Send.tsx:338]: that makes sense! i was thinking that the `Send` screen should stay as simple as possible too. thanks!

File: packages/react-components/components/WithPasteAware.tsx (hunk 1 above)
Comment (michellewong793) [reply packages/react-components/components/WithPasteAware.tsx]: removed!

File: packages/mobile/src/fiatExchanges/__snapshots__/ProviderOptionsScreen.test.tsx.snap [hunk 3]
```diff
@@ -183,7 +183,7 @@ exports[`ProviderOptionsScreen renders correctly 2`] = `
                     "justifyContent": "space-between",
                   }
                 }
-                testID="Provider/Transak"
```
[the remark this answers is not in the document]
Comment (michellewong793) [reply packages/mobile/src/fiatExchanges/__snapshots__/ProviderOptionsScreen.test.tsx.snap:192]: oops, i think i accidentally deleted this when merging with main + i had new snapshots

File: packages/mobile/src/send/Send.tsx [hunk 4]
```diff
@@ -357,6 +357,9 @@ const styles = StyleSheet.create({
   headerRightContainer: {
     paddingRight: 16,
   },
+  leftIcon: {
```
Comment (gnardini) [inline packages/mobile/src/send/Send.tsx:360]: This is not used anymore, so we can delete it

**Review by gnardini — APPROVED** [review]
One final nit, everything else looks good, great job on this!

File: packages/mobile/src/send/Send.tsx (hunk 4 above)
Comment (michellewong793) [reply packages/mobile/src/send/Send.tsx]: thanks :) done!

**Comment by Celoqa** [comment]
Hi @gnardini  I have verified this issue using latest released Android internal build V1.14.0 (1004294350) & Test Flight build V1.14.0(58) & observed following:
**Observations:**

- [x] It is observed that, user able to see "To" text instead of Magnifying glass icon.

- [x] Also it is observed that, "Name, phone, or address" Search field place holder displayed
- [ ] For now, we have not observed "Get up to $5 by referring friends" text below search field. Will update you when we get screen.

- [ ] When user enters phone number from Contact list then it is observed that only that saved Contact name displayed without phone number as well as "Send to Phone Number" text [Attachment](https://drive.google.com/file/d/1TGUjaJr0_b5OCR0SsXr9pzxaccjlm0ZP/view?usp=sharing)

- [ ] When user enters valora user's contact (other than phone Contact list)then only  "Send to Phone Number" text displayed without phone number & searched user's name. [Attachment](https://drive.google.com/file/d/1zT1_CV08IgiC7OfCy2AzSnCnuHWnBD3_/view?usp=sharing)

- [ ] Also it is observed that, user is able to see Account number twice one as name & one as address number with valora symbol (V) when user search account address of user B.We have not observed "Send to Address" text [Attachment](https://drive.google.com/file/d/1XjNG6rFQl-VV9JI-PklhWsCz-DMHJ-z1/view?usp=sharing)
Can you please let us know needful for above unchecked points. Are they expected or we need to raise tickets for same ?

**Verified On Devices:** iPhone SE (14.2), Redmi Note 8 (9.0)

Thanks..!!

**Comment by Celoqa** [comment]
Hey @gnardini Is there any update on above observations ?

**Comment by gnardini** [comment]
Everything looks expected! Thanks!

**Pull request merged**